### PR TITLE
Davenport Hill III - Intro to Git Assignment 10/7

### PR DESCRIPTION
**Pull request by dh2540**
I changed my code based on my partner Cole's suggestion. Here is the advice I didn't take and the reasons.
1. CAG: I would personally add a space between each doc block and the code, but that is personal preference
Personally, having a line between the code and comments would ruin the immersion of the program. I think it is better to have no blank lines during comment to code transitions. 
2. CAG: I Think the program in the IDE and in Codechat both read very well. I might would consider adding a resources section at the bottom but that again is just preference. The main change I would make would be to put your code into the given code and present it as one cohesive program.
I added a resource section at the bottom, which was a pleasant call on him. This will help any regular person understand certain parts of my code more. The part that I disagreed with was putting my code with the given code. I did not make all of this code. That is why I made that distinction.

### CODE REVIEW

File: 2nd commit.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+2nd commit to new branch
```
Comment (bjones1) [inline 2nd commit.txt:1]: Remove all these misc files.

File: thorgame/dh2540/The Power of Thor.cpp [hunk 2]
```diff
@@ -38,47 +44,51 @@ using namespace std;
...
         int remaining_turns; // The remaining amount of turns Thor can move. Do not remove this line.
         cin >> remaining_turns; cin.ignore();
 
-        // Write an action using cout. DON'T FORGET THE "<< endl"
-        // To debug: cerr << "Debug messages..." << endl;
-
-        // ## *My Code*
-        // If Thor's x and y coordinates are greater than the light's x and y coordinates,
-        // move Thor northwest by one.
+        // Write an action using cout. DON'T FORGET THE "<< endl" To debug: cerr
```
Comment (bjones1) [inline thorgame/dh2540/The Power of Thor.cpp:61]: This doesn't fit the rest of the comment you made -- remove or revise. I assume it was provided by the codingame site?

File: thorgame/dh2540/The Power of Thor.cpp [hunk 3]
```diff
@@ -117,4 +127,10 @@ int main() {
 
         }
     }
-}
\ No newline at end of file
+}
+
+// Helpful Links
```
Comment (bjones1) [inline thorgame/dh2540/The Power of Thor.cpp:132]: Make this a numbered or bulleted list.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by dh2540** [comment]
I made all the changes you wanted. I got rid of the comments the CodinGame site provided, made a list for the website, etc.

**Review by bjones1 — APPROVED** [review]

**Pull request merged**